### ISSUE TITLE
unrecognized arguments: Alien'

### ISSUE THREAD

**Issue by Ustinian-jt** [Stilwell-Git/Doubly-Bounded-Q-Learning#1]
sorry.when I run the commond "python train.py --tag='DB-ADP Alien' --env=Alien",I get this error:
`usage: train.py [-h] [--tag TAG] [--env ENV] [--alg {dqn,cddqn}]
                [--learn {atari,atari_dbadp}] [--gamma GAMMA]
                [--sticky STICKY] [--noop NOOP] [--frames FRAMES]
                [--rews_scale REWS_SCALE] [--test_eps TEST_EPS]
                [--epochs EPOCHS] [--cycles CYCLES] [--iterations ITERATIONS]
                [--timesteps TIMESTEPS] [--test_rollouts TEST_ROLLOUTS]
                [--test_timesteps TEST_TIMESTEPS] [--save_rews SAVE_REWS]
                [--buffer {framestack}] [--buffer_size BUFFER_SIZE]
                [--batch_size BATCH_SIZE] [--warmup WARMUP]
                [--train_batches TRAIN_BATCHES] [--train_target TRAIN_TARGET]
                [--eps_l EPS_L] [--eps_r EPS_R] [--eps_decay EPS_DECAY]
                [--optimizer {adam,rmsprop}] [--q_lr Q_LR]
                [--Adam_eps ADAM_EPS] [--nstep NSTEP] [--double DOUBLE]
                [--dueling DUELING]
train.py: error: unrecognized arguments: Alien'
`
How can I solve it，thank you~

**Comment by Stilwell-Git** [comment]
This issue may be caused by a Windows-style CRLF character (i.e., '\r') at the end of the command line.
Please check the line ending of the command script and ensure it uses a Unix-style LF ending.
If the issue is not solved, do not hesitate to contact us, and we will try to figure it out.

**Comment by Ustinian-jt** [comment]
Thank you for your reply，I have checked my commond,the problem still seems to be unsolved......
![image](https://user-images.githubusercontent.com/43819608/201557158-ab361253-9809-4399-bd20-f55521314050.png)

**Comment by Stilwell-Git** [comment]
Could you please try this command?
```
python train.py --tag='DB-ADP-Alien' --env=Alien
```

**Comment by Ustinian-jt** [comment]
emmmmm,it seems turned out to be this
![image](https://user-images.githubusercontent.com/43819608/201560284-3b599f86-7103-4e80-9230-d25dd8f232b8.png)

**Comment by Stilwell-Git** [comment]
We update the code slightly. Please `git pull` and try again.
There might be some issues running this code in Windows. It would be better to use a Linux system.

**Comment by Ustinian-jt** [comment]
Thank you very much,Now,I use the Ubuntu16.04 to test the code,it turned out to be this:
![image](https://user-images.githubusercontent.com/43819608/201675025-f9f2b84d-1791-4008-b1a5-0b17dc74ccce.png)
Did I succeed？ How can I view the effect of the experiment? it seems to  at a standstill.

**Comment by Stilwell-Git** [comment]
Yes. Please wait for a while. The log will be printed in every 50000 environment steps.

**Comment by Ustinian-jt** [comment]
OMG,Something bad happened again.....
![image](https://user-images.githubusercontent.com/43819608/201677639-072744e5-fb8b-43cb-888d-ea804a799de2.png)

**Comment by Stilwell-Git** [comment]
We figure out a typo bug. Please `git pull`, and the issue should be solved.

**Comment by Ustinian-jt** [comment]
Thank you very much~

**Comment by Ustinian-jt** [comment]
Sorry,recently I run this code，I found that I could not complete the whole code training process.When the program iterated 350000 times, it would be automatically terminated.Is there somthing wrong with the code？I'm sorry to bother you again
![image](https://user-images.githubusercontent.com/43819608/203931022-a7b9efae-a42a-4c55-a7fe-48047bf32c9c.png)

**Comment by Ustinian-jt** [comment]
Are you from China？Can we chat with wechat?If you don't mind,I can pay you some money，thank you~
